dcrjson/chainsvrresults.go

```diff
@@ -29,6 +29,7 @@ type GetBlockHeaderVerboseResult struct {
 	Size          uint32  `json:"size"`
 	Time          int64   `json:"time"`
 	Nonce         uint32  `json:"nonce"`
+	StakeVersion  uint32  `json:"stakeversion,omitempty"` // XXX should we omitempty?
```

Comment:
I see no reason to `omitempty`.  Old blocks will just show up as either 0, or whatever happened to be in the extra data at those bytes.